scripts/glutton/update/src/main.rs

```diff
@@ -0,0 +1,176 @@
+use crate::glutton_para::runtime_types::sp_arithmetic::per_things::Perbill;
+use clap::{Parser, ValueEnum};
+use subxt::{
+	dynamic::Value,
+	ext::sp_core::{sr25519::Pair as Sr25519Pair, Pair},
+	tx::PairSigner,
+	utils::AccountId32,
+	OnlineClient,
+};
+
+mod config;
+use crate::config::GluttonConfig;
+
+#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, ValueEnum)]
+enum RelayChain {
+	Local,
+	Kusama,
+	Versi,
+}
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+struct Cli {
+	/// Relay chain context selection: kusama, versi or local
+	#[clap(value_enum)]
+	#[arg(short, long)]
+	relay: RelayChain,
+
+	/// Update Glutton Parachains from 'para_id = from' (inclusive)
+	#[arg(short, long)]
+	from: u32,
+
+	/// Update Glutton Parachains to 'para_id = to' (inclusive)
+	#[arg(short, long)]
+	to: u32,
+
+	/// Optional new 'storage' value to update
+	/// Value in parts per billion. E.g: 500000000 equals 50% of a block `ref_time` consumed
+	#[arg(short, long)]
+	storage: Option<u32>,
+
+	/// Optional new 'compute' value to update
+	/// Value in parts per billion. E.g: 500000000 equals 50% of a block `proof_size` consumed
+	#[arg(short, long)]
+	compute: Option<u32>,
+
+	/// Sudo account secret seed in hex format
+	#[arg(short, long)]
+	private_key: String,
+}
+
+// Generate an interface that we can use from the node's metadata.
+#[subxt::subxt(runtime_metadata_path = "./artifacts/glutton_metadata.scale")]
+pub mod glutton_para {}
+
+type RuntimeCall = glutton_para::runtime_types::glutton_runtime::RuntimeCall;
+type GluttonCall = glutton_para::runtime_types::pallet_glutton::pallet::Call;
+
+async fn get_account_nonce(
+	client: &OnlineClient<GluttonConfig>,
+	account: &AccountId32,
+) -> Result<u32, Box<dyn std::error::Error>> {
+	let runtime_api_call = subxt::dynamic::runtime_api_call(
+		"AccountNonceApi",
+		"account_nonce",
+		vec![Value::from_bytes(account)],
+	);
+
+	let nonce = client
+		.runtime_api()
+		.at_latest()
+		.await?
+		.call(runtime_api_call)
```

Comment:
If you want to avoid the unstable-metadata thing you could also use (assuming nonce is a u32, which is true in substrate/polkadot and I'd expect most other chains):

```rust
let next_nonce: u32 = client
    .runtime_api()
    .at_latest()
    .await?
    .call_raw("AccountNonceApi_account_nonce", Some(&account.encode()))
    .await?;
```

I'm actually going to move Subxt to do this too, and will likely expose a way to get this in the next version